eva_submission/xlsx/xlsx_validation.py

```diff
@@ -153,8 +153,20 @@ def check_date(self, row, key, required=True):
         if required and key not in row:
             self.error_list.append(f'In row {row.get("row_num")}, {key} is required and missing')
             return
-        if key in row and (isinstance(row[key], datetime) or str(row[key]).lower() in not_provided_check_list):
+        if key in row and (
+                isinstance(row[key], datetime.date) or
+                isinstance(row[key], datetime.datetime) or
+                self._check_date_str_format(row[key]) or
+                str(row[key]).lower() in not_provided_check_list
+        ):
             return
         self.error_list.append(f'In row {row.get("row_num")}, {key} is not a date or "not provided": '
                                f'it is set to "{row.get(key)}"')
 
+    def _check_date_str_format(self, d):
+        try:
+            datetime.datetime.strptime(d, "%Y-%m-%d")
```

Comment:
I am wondering why we are making an exception to accomodate just this format... what happens if dates are provided in another format?

Reply:
I'm actually not convinced we need this. 
This would only be triggered if the spreadsheet it populated with date forced into string in Excel. 
The rational here is that we state the date should be formatted `YYYY-MM-DD` so that's why I only added support for this format.
In most cases the dates will be provided as Excel formatted dates which should be converted to python `datetime.date` or `datetime.datetime`